chains/evm/calls/bridge.go

```diff
@@ -401,3 +401,65 @@ func Withdraw(client ClientDispatcher, txFabric TxFabric, gasPricer GasPricer, g
 
 	return &h, nil
 }
+
+// public function to pause: deposits, deposit executions,
```

Comment:
https://go.dev/blog/godoc

Usually, godocs should start with the name of the function and be imperative (Pause pauses: deposits ... ).
Public is obvious as it is uppercased.

Reply:
Yep, good point, will fix. At some point we'll probably need to re-document all of our code so that it properly adheres to godoc specs